xds/src/test/java/io/grpc/xds/CommonBootstrapperTestUtils.java

```diff
@@ -88,7 +88,7 @@ public static Bootstrapper.BootstrapInfo buildBootstrapInfo(
       String certInstanceName1, @Nullable String privateKey1,
       @Nullable String cert1,
       @Nullable String trustCa1, String certInstanceName2, String privateKey2, String cert2,
-      String trustCa2) {
+      String trustCa2, @Nullable String spiffe) {
```

Comment:
nit: Can we use a more descriptive name than "spiffe"? It is ambiguous whether this refers to a SPIFFE cert or something else. For example, "spiffeTrustMap"?